docs/adr/0012-moc-infra-cluster-deployment-methods.md

```diff
@@ -0,0 +1,47 @@
+# Cluster deployment methods for moc-infra cluster
+
+## Context and Problem Statement
+
+This document suggests options for the preferred deployment method for the **moc-infra cluster** at the MOC(Mass Open Cloud). This hub cluster will be the central openshift cluster to manage and deploy other clusters on-demand.
+
+The moc-infra cluster is a bare-metal cluster, deploying an OpenShift Container Platform on bare metal can be achieved by several methodologies.
+
+## Considered Options
+
+### UPI - User provioned indrastructure
+
+Deploy an OpenShift cluster on infrastructure that you prepare and maintain.
+
+We decided not to take this option under consideration since this method includes heavier infrastructure configuration.
```

Comment:
I think the adjective "heavier" here is a little unclear. Maybe:

```suggestion
We decided not to take this option under consideration since there is more effort involved in setting up the infrastructure.
```